backend/blueprint/api/controllers/homePage.py

```diff
@@ -4,7 +4,7 @@
 router = Blueprint('homePage', __name__)
 
 
-@router.route('/', methods=['POST'])
+@router.route('/', methods=['GET', 'POST'])
```

Comment:
Can you please explain why we need `GET` here?

Reply:
I added `GET` to test, removing 👍